stob/ad.c

```diff
@@ -1703,9 +1720,9 @@ static int stob_ad_write_map_ext(struct m0_stob_io *io,
 			/* handle extent deletion. */
 			if (adom->sad_overwrite) {
 				M0_LOG(M0_DEBUG, "del: val=0x%llx",
-					(unsigned long long)seg->ee_val);
+				       (unsigned long long)seg->ee_val);
```

Comment:
This change is not required for this patch. The same could be applied to the changes below.